flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSource.java

```diff
@@ -322,14 +323,28 @@ private void processMessage(Delivery delivery, RMQCollectorImpl collector) throw
         deliveryDeserializer.deserialize(envelope, properties, body, collector);
     }
 
+    /**
+     * Returns the next message delivery timeout used in the queueing consumer. If not specified in
+     * the provided connection config, the default value of 30000 milliseconds will be returned.
+     *
+     * @return consumer delivery timeout in milliseconds
+     */
+    protected int getDeliveryTimeout() {
+        return rmqConnectionConfig.getDeliveryTimeout().orElse(DEFAULT_DELIVERY_TIMEOUT);
```

Comment:
Should this default value for delivery timeout put into the constructor of `RMQConnectionConfig` ?

Reply:
Thanks. I've moved the default value setting to `RMQConnectionConfig` (14571a9)